src/Chapter3.hs

```diff
@@ -374,6 +380,19 @@ after the fight. The battle has the following possible outcomes:
 
 -}
 
+
+data Properties = P Int Int Int
```

Comment:
It's generally recommended to use records instead of unnamed fields. When I see three `Int`s, it's not clear from the data type definition what they mean. But if you implement this data type as a record, its definition already becomes clearer, even if you don't use record fields as getters or setters :slightly_smiling_face: 

Alternatively, you can write documentation to each field, but in such cases it almost always makes sense to implement data types with a single constructor as records.